lua/laravel/artisan.lua

```diff
@@ -19,16 +19,18 @@ end
 ---@param opts table | nil
 ---@return table, boolean
 artisan.run = function(cmd, runner, opts)
+  local laravel = require("laravel").app
+
   local command = cmd[1]
   opts = opts or {}
   runner = runner
-    or require("laravel").app.options.commands_runner[cmd[1]]
-    or require("laravel").app.options.default_runner
+    or laravel.options.commands_runner[cmd[1]]
+    or laravel.options.default_runner
 
   table.insert(cmd, 1, "artisan")
 
-  local ok = require("laravel").app.if_uses_sail(function()
-    table.insert(cmd, 1, "vendor/bin/sail")
+  local ok = laravel.if_uses_sail(function()
+    cmd = laravel.buildCmd(laravel.options.exec.artisan, cmd)
```

Comment:
The "vendor/bin/sail" should be remove, but the option exec artisan does not make sense since is the sail command not artisan, also should remove the php. this should be constants at some new property `runnables` or something similar.

Reply:
so the reason for that here is that i think the artisan command for sail is `sail artisan command` however to execute it in container, you would need `docker-compose exec app *php* command`, and if you want to execute the php command in the sail container you still need to add php `sail php script.php`. We certainly can just append "php" string to the sail exec as well. It does give a user a bit more freedom in settings though.

Reply:
Agree always appending `php` will minimize the differences.

Reply:
the only edge case i can think of is if you want to run some command on container itself, but I am not sure this plugin should handle that.

Reply:
@adalessa if we want to use sail commands, like `up`, `down` etc, the sail may need to be `docker-compose` without `exec`

Reply:
Yes, that I have been thinking, this is more like environment commands. I still thinkin on how to define it. I think something like the environment could define the commands, and sail will define their own, and docker compose the same.